README.md

```diff
@@ -38,19 +38,33 @@ Edit `.env` and setup your own config
 # Make sure your node version is `v12.*`, as `v14.*` is not yet supported by ganache
 yarn run-ganache
 
-# Clone dex-contracts project (in another tab)
+# Clone dex-liquidity-provision project (in another tab)
+#   It doesn't matter where you clone the project, this project is independent from dex-subgraph
+git clone https://github.com/gnosis/dex-liquidity-provision
```

Comment:
It's starting to get complicated with so many dependencies.
If we can make it not a strict dependency, I think is best. Otherwise, I think we should look for ways on simplifying the setup.

If indexing the CMM is optional, we can add a section in the readme for it, and leave in the initial setup just dex-contracts

Reply:
You are right that it is getting more and more complex.

If you are okay with it, I would more: 
https://github.com/gnosis/dex-contracts/blob/master/scripts/ganache/setup_thegraph_data.ts

into the dex-liquidity-provision repo and then we can only rely on dex-liquidity-provision. We can also do all the migration from there etc.

Wdyt?

Reply:
I found a smooth method for running the same script from within dex-liquidity-provision. this solves this issue